packages/drivers/odsp-driver/src/odspDocumentService.ts

```diff
@@ -251,6 +251,8 @@ export class OdspDocumentService implements IDocumentService {
      * @returns returns the document delta stream service for onedrive/sharepoint driver.
      */
     public async connectToDeltaStream(client: IClient): Promise<IDocumentDeltaConnection> {
+        assert(this.currentConnection === undefined || this.currentConnection.disposed,
```

Comment:
I think disposed => no connection, no?
Should we enforce that? Either through assert (i.e. push responsibility to callers) or through actions here (i.e. close connection on dispose)? I like the later, as it's more generic - it does not matter what happened before, but once dispose() is called, we ensure all resources are closed.